docs/TrialAccounts/offboarding.md

```diff
@@ -0,0 +1,180 @@
+//    key added to the newly created account.     
+///
+/// Returns `true` if the claim was successful meaning the `new_account_id` was 
+/// created and all assets were sent to it.
+function create_account_and_claim(new_account_id: string, new_public_key: string) -> Promise<boolean>
+```
+
+For an arbitrary linkdrop (a URL containing the contract and secret key), the flow is as follows:
+1. The user clicks a link or scans a QR code representing the linkdrop.
+2. The app calls `get_key_information` to check what's in the linkdrop.
+    - If the linkdrop was already claimed (or doesn't exist), the call will panic and the frontend indicates that the drop is invalid or claimed.
+3. Depending on what is returned from the call, the app should display what assets the user is about to claim (e.g NFTs, FTs, $NEAR etc.)
+4. The frontend exposes a form field and the user either inputs a new account or an existing one.
+5. The frontend calls `claim` or `create_account_and_claim` depending on the user's input. 
+6. The linkdrop is claimed and the assets are sent to the user's account.
+
+### Trial Account Linkdrops
+
+Now that the basics of linkdrop claiming have been covered, you'll look at how trial accounts can be treated as a linkdrop. From above, there are a few requirements that need to be met:
+- The URL should contain a contract and secret key
+- The contract should implement the `get_key_information` method so that frontends know when a drop is valid.
+- The contract should implement the `create_account_and_claim` method for claiming.
+
+The `new_public_key` that is passed into `create_account_and_claim` is added as the account's full access key and is generated on a per-app basis. For example, MyNEARWallet generates the key using a seedphrase while FastAuth generates it using biometrics. The trial account contract can use `create_account_and_claim` to perform the offboarding logic. It should take in the new public key and do the following (assuming the account can exit):
+1. Delete the limited access key used during the trial.
+2. Repay the funder for any required $NEAR
+3. Delete any state that is being used by the contract
+4. Delete the contract itself and free up all the storage
+5. Create a new access key with the `new_public_key` and give it full access to the account.
+
+You'll notice that in the example above, the only parameter that was used was `new_public_key` but in the official linkdrop standard, the `create_account_and_claim` function takes two parameters:
+
+```ts
+function create_account_and_claim(new_account_id: string, new_public_key: string)
+```
+
+This is because for regular linkdrops, it is assumed that by calling `create_account_and_claim`, the account is being created from scratch and the public key is being added as full access. In the case of trial accounts, the account *already exists* and we're simply adding the passed in public key as full access.
```

Comment:
does this mean that our modified `create_account_and_claim` will only require the FAK arg? could a quick code snippet for this be shown?